docs/mgmt-cluster/01_install_tkg_mgmt.md

```diff
@@ -1,20 +1,20 @@
 # Install TKG Management Cluster
 
-Follow the [official docs](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/index.html) for background and pre-requisite tasks.
+Follow the [official docs](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.1/vmware-tanzu-kubernetes-grid-11/GUID-index.html) for background and pre-requisite tasks.
```

Comment:
Pros and cons on that change:
- Pros: It's a deeper link and ensures the user will reach that part
- Cons: It's version specific and will have to be updated version after version.
Let's discuss.

Reply:
I had the same concern, however, the original link really requires you to make two additional clicks to get the the desired page.  First have to click on "Tanzu Kubernetes Grid"  and then again on "Installing Tanzu Kuberentes Grid".  In the end, I know we have other specific references to 1.1 so I figured it would be better to provide more clarity and take the burden of having to bump references when new releases come out.